### PR TITLE
new records

### PR DESCRIPTION
**Pull request by thea-m**
I'm back! 😄

### CODE REVIEW

File: new/PRS13549SardaD.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13549SardaD" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Sarḍa Dǝngǝl</title>
```
Comment (MassimoVilla) [inline new/PRS13549SardaD.xml:8]: Śarḍa

File: new/PRS13549SardaD.xml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        <revisionDesc>
+            <change who="DR" when="2021-09-01">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ሠርፀ፡ ድንግል፡</persName>
+                    <persName xml:lang="gez" corresp="#n1" type="normalized">Sarḍa Dǝngǝl</persName>
```
Comment (MassimoVilla) [inline new/PRS13549SardaD.xml:49]: Śarḍa

**Review by MassimoVilla — CHANGES_REQUESTED** [review]
Thank you Thea! :)

**Comment by thea-m** [comment]
Thank you!

**Comment by thea-m** [comment]
Can this be merged?

**Review by MassimoVilla — APPROVED** [review]

**Pull request merged**